doc/invites.md

```diff
@@ -0,0 +1,21 @@
+Guidelines for Instruqt invites
+===============================
+
+* Public title: `Replicated Hands-On Labs for [Vendor Name]`
+* Public description: `An opportunity for the [Vendor Name] team to explore the features and capabilities of Replicated in a lab environment.`
+* Internal description: ok to leave empty
+* Select tracks: 
+    * Select the tracks you want the Vendor to have access to. 
+    * Do not include tracks that are not relevant. For example, if they will not use Helm, do not include any Helm related tracks. 
+    * Also, order is important!
+* Content Restrictions:
+    * How long is content available to users after they access the invite?: Select 1 month.
+    * How many times can each track be played per person?: 0 is fine. (unlimited)
+* Invite Restrictions:
+    * When will this invite stop accepting new users?: Select last day of the following month.
+    * How many unique users can access this invite? 0 is fine. (unlimited)
+* Access settings:
+    * Allow anonymous access: false
+    * Allowed email addresses: domain based access! `[Vendor Name].com` or `[Vendor Name].ai` or ...
```

Comment:
Add `replicated.com` as well? or the individual emails for the account team? I've been doing that, since I know the AEs are going to follow the link when the email goes out.

Reply:
Interesting thought. What happens if you get multiple invites for the same email? You see the tracks multiple times?
We could consider this as a follow-up PR.